tests/nemo_text_processing/en/test_sparrowhawk_normalization.sh

```diff
@@ -11,7 +11,10 @@ runtest () {
   while read testcase; do
     IFS='~' read written spoken <<< $testcase
     # replace non breaking space with breaking space
+    # Use below if postprocessor is not used. Comment if it is used
     denorm_pred=$(echo $written | normalizer_main --config=sparrowhawk_configuration.ascii_proto 2>&1 | tail -n 1 | sed 's/\xC2\xA0/ /g')
+    # Use below if postprocessor is  used. Comment if it is not used
```

Comment:
maybe we should have a flag to control this?

Reply:
We could consider checking if the far file exists for postprocessor and then choose the corresponding proto. However It would be better if we explicitly document this in https://github.com/NVIDIA/NeMo-text-processing/tree/main/tools/text_processing_deployment

IMHO there really isn't a need to have a command-line flag for this

Reply:
ok, documentation makes sense. Could you add it as part of this PR?should the version with the postprocessor be the default?

Reply:
Will add documentation. For English, version with PP could be the default. However need to change update the test cases. Should this be done with this PR or a separate PR

Reply:
let's do test cases separately to merge this PR since [PR112](https://github.com/NVIDIA/NeMo-text-processing/pull/112) relies on the new postprocessor